processing/src/main/java/io/druid/query/Query.java

```diff
@@ -70,8 +70,18 @@
 
   String getType();
 
+  /**
+   * @deprecated use {@link QueryPlus#run(QuerySegmentWalker, Map)} instead. This method could be removed in the next
```

Comment:
Could you add a comment describing this plan? Otherwise, anyone reading the code might be confused (like I was) about what the replacement is for this deprecated method that has no obvious replacement.